### PR TITLE
Remove social icons fix #34

### LINKED ISSUE #34: Remove social icons
We don’t have any social media accounts. Maybe we should comment out the icons till we get them? 
Bottom of homepage is where they're at.

### PR DESCRIPTION
**Pull request by AustinThornley**
Fixes issue #34

### CODE REVIEW

File: pages/main.html [hunk 1]
```diff
@@ -1,70 +1,54 @@
...
-    scrolling="no"
-  ></iframe>
-  <nav class="social-media-links flex justify-center space-x-4">
-    <img src="./assets/images/facebook.png" class="w-10 h-10" />
-    <!-- <img src="./images/messenger.svg" /> -->
-    <img src="./assets/images/instagram.png" class="w-10 h-10" />
-    <img src="./assets/images/twitter.png" class="w-10 h-10" />
-  </nav>
+	<div class="flex-1 shrink"></div>
+	<div class="basis-1/2 flex flex-col flex-grow items-center text-center align-middle">
```
Comment (AustinThornley) [inline pages/main.html:3]: Ya' know, I really tried to get a pretty looking git diff but I think someone must have special tab/space rules that none of my editors have. Haha oops! 😅

File: pages/main.html [hunk 2]
```diff
@@ -1,70 +1,54 @@
 <main class="flex flex-col justify-between h-screen py-2" id="main">
```
Comment (AustinThornley) [inline pages/main.html:1]: Ya' know, I really tried to get a pretty looking git diff but I think someone must have special tab/space rules that none of my editors have. Haha oops! 😅 a default formatter and format config file would help with that, it's super helpful on open source projects.

Comment (fso5) [reply pages/main.html:1]: good thinking! go ahead and add one and if anyone has issues with it we can work out specific formatting ideas.

**Review by fso5 — APPROVED** [review]

File: pages/main.html (hunk 2 above)
Comment (AustinThornley) [reply pages/main.html:1]: Does everyone have Prettier or use an editor that could get prettier? I know VSCode has it

**Comment by AustinThornley** [comment]
Also I’ll leave one of you to merge this whenever ready @NikhilMath @bojoki

File: pages/main.html (hunk 2 above)
Comment (fso5) [reply pages/main.html:1]: Ya I think that should be fine, you could make the changes. 

Concerning the merge, Nikhil recently made it so once this is approved it should be merged automatically, you could optionally remove him as an approver, or nudge him to approve. I mean I could manually merge it, but that would circumvent the auto merge thing, which then would defeat the purpose of having it in place.

tldr, hopefully nikhil pops in.

**Comment by NikhilMath** [comment]
Hey @AustinThornley 
I completed deleted everything, so I wont be merging this haha. but thank you so much for putting in the effort!!
